tailcfg/tailcfg.go

```diff
@@ -1797,6 +1801,10 @@ type PeerChange struct {
 	// Key, if non-nil, means that the NodeID's wireguard public key changed.
 	Key *key.NodePublic `json:",omitempty"`
 
+	// KeySignature, if non-nil, means that the signature of the wireguard
+	// public key has changed.
+	KeySignature tkatype.MarshaledSignature `json:",omitempty"`
```

Comment:
Don't add to PeerChange now. This would need to be in its own change with a capver bump.

Reply:
Will do!

Why is it unsafe without masking it behind a capver? I would have thought it had no effect when unset.

Reply:
It's just kinda confusing in the history, why there's a no-op unused field.